doc/quick-start-user-guide.md

```diff
@@ -9,19 +9,26 @@ This guide takes the following as an example of IPs for the two nodes:
  * **Node 1**: `192.168.56.101`
  * **Node 2**: `192.168.56.102`
 
-Please follow these steps for setup on **each of the two nodes**.
+Please follow these steps for setting up glusterd2 on **each of the two nodes**.
 
 ### Installing dependent packages
 
 Install rpcbind:
 
+For CentOS: 
+```sh
+# yum install rpcbind
+```
+For other distors please install rpcbind and then follow the steps given below.
```

Comment:
rpcbind is no longer needed for glusterd2

Reply:
@prashanthpai I had a discussion with @kshlm regarding same, as he suggested  I needed to check with others before removing it. Will remove it. and update the PR